### DISCUSSION TITLE
发个图

### DISCUSSION THREAD

**Discussion by daofeng9612** [share121/inter-knot#2328] [其他]
发个图

**Comment by daofeng9612** [comment]
![1721851201327785780a4670c2770f00e709bb6dea3d52ce6](https://github.com/user-attachments/assets/33709ea3-4a85-4378-b571-be5389b82702)

**Comment by nightrain2333** [comment]
这清晰度看爽了<img src="https://img-static.mihoyo.com/communityweb/upload/81ed7cd6675af50aec9cc622bda99b88.png" alt="比利_害羞" width="75" >
